lib/integrations/awsoidc/eks_enroll_clusters_test.go

```diff
@@ -229,6 +235,33 @@ func TestEnrollEKSClusters(t *testing.T) {
 					`can't enroll EKS cluster "EKS1" - expected "ACTIVE" state, got "PENDING".`)
 			},
 		},
+		{
+			name:         "private cluster in cloud is not enrolled",
+			enrollClient: baseClient,
+			eksClusters: []eksTypes.Cluster{
+				{
+					Name: aws.String("EKS3"),
+					ResourcesVpcConfig: &eksTypes.VpcConfigResponse{
+						EndpointPrivateAccess: true,
```

Comment:
I would either remove the configuration to rely on the default value or be explicit about `EndpointPublicAccess` being set to false.
Mentioning another field here may cause confusion about what value we are looking at production code.

Reply:
Yep, changed to be explicitly false.